### ISSUE TITLE
Suggestion

### ISSUE THREAD

**Issue by shaikhnedab** [Summer-16/CSGO-VMPanel#4]
Add a checkbox menu to add VIP on multiple server at once like adding admins using sourceban webpanel.
Right now we have to repeat the vip adding procedure if we want to add one vip to multiple server.

**Comment by Summer-16** [comment]
Points suggested by @nguyenbaodanh 
- users can log in via steam to buy VIP or check days left for renewal if they want.
- integrate with sourcebans++ ( including admin groups, flags, server groups, etc...)
- automatic VIP/admin payment through pay gates like Paypal, Skrill and more (including automatic monthly renew)
- users can choose server groups to register for VIP/Admin subscribe
- or users can choose a single server to subscribe
- VIP/ admins add/expire log?
- Users expire / days left alert?
- log server kick/ban/gag/mute for admins
- coupon codes/discount codes?
- Servers remote console for admins?

**Comment by maxz1n** [comment]
Dear is a pleasure to greet you I want first to congratulate you for the great plugins CSGO Vip Management Panel which I have no idea how to install it currently I have a dedicated server on windows server can you help me install it please! my discord is: sokka#7685

**Comment by Tetragromaton** [comment]
What about API for Sourcemod plugins ?

**Comment by crashzk** [comment]
Any possibility of adding new payment methods?

- **PagSeguro -** https://pagseguro.uol.com.br/
- **Mercado Pago -** https://www.mercadopago.com.br/

In my country these are the most used forms of payment as well.

**Comment by Summer-16** [comment]
For now PayPal will be the primary payment option, as it is accepted internationally.
Although in parallel i am working on a secondary payment option which will include credit cards and all

**Comment by crashzk** [comment]
> * coupon codes/discount codes?

Yes, I saw that you intend, perhaps to implement discount coupons. I do not know if you think along with this to make coupons for access to privileged players.

That is, instead of a discount the coupon would give him privileges. It would be a good idea too.

**Comment by Summer-16** [comment]
Currently, I just finished adding a new payment gateway PayU Money which will be including all the wallets and UPI options requested by Indian members,
After that, I will be looking into test the VIP feature

**Comment by crashzk** [comment]
Any news about the Mercado Pago payment method?

In addition to it, it would be interesting for people from Latin America to accept PagSeguro, MercadoPago, PicPay.

It would even be interesting to see a way for the Panel's ADMIN to put the payment options he would like, something similar to what WHMCS offers, modules and such.
